### ISSUE TITLE
Errors during build with OpenCover

### ISSUE THREAD

**Issue by Alessar** [alhardy/AppMetrics.Reporters#56]
`.\build.ps1 --ScriptArgs '-CoverWith=OpenCover'`

Expecting: All is Ok. Coverage is done.

Actual: Errors happened.
```
Committing...
No results, this could be for a number of reasons. The most common reasons are:
    1) missing PDBs for the assemblies that match the filter please review the
    output file and refer to the Usage guide (Usage.rtf) about filters.
    2) the profiler may not be registered correctly, please refer to the Usage
    guide and the -register switch.
Loading coverage file ./artifacts/coverage/openCoverCoverage.xml
Failed to load coverage file ./artifacts/coverage/openCoverCoverage.xml
System.InvalidOperationException: Sequence contains no elements
   at System.Linq.Enumerable.Max[TSource](IEnumerable`1 source)
   at OpenCover.Framework.Model.InstrumentationPoint.ResetAfterLoading() in c:\projects\opencover\main\OpenCover.Framework\Model\InstrumentationPoint.cs:line 38
   at OpenCover.Framework.Persistance.BasePersistance.ReassignCoverageSession(CoverageSession session) in c:\projects\opencover\main\OpenCover.Framework\Persistance\BasePersistance.cs:line 127
   at OpenCover.Framework.Persistance.FilePersistance.LoadCoverageFile() in c:\projects\opencover\main\OpenCover.Framework\Persistance\FilePersistance.cs:line 92
```

**Comment by alhardy** [comment]
Might be because the debug type set in `common.props` is `embedded`, this was required to use source link from memory

**Comment by Alessar** [comment]
Looks like yes. Valid values for `DebugType` are `full`, `pdbonly` and `none`.

**Comment by alhardy** [comment]
hmm last I knew `embeded` was valid. OpenCover required debug type `full` though in which case source link doesn't (or at least didn't) work. 

Build script also supports DotCover which works either way

**Comment by Alessar** [comment]
I tested localy. I did not see error like in current #56 issue when use any of `full,pdbonly,none`. So any of that variables are good. Maybe `embedded` is good option. But there is issue in this case. Maybe it related with some specific version or environment?
An also I want to notice that coverage results in that values are something different from each others.

**Comment by alhardy** [comment]
There should only be slight differences, would also depend on the type of coverage you're looking at e.g. branch vs line

**Comment by Alessar** [comment]
Yes. There are differences in branches and in lines. Maybe it is related with percent calculation with different amount of code which available in different build types. I only wanted to make notice about it.

**Comment by Alessar** [comment]
PR #59 fixed current #56 issue.
